### DISCUSSION TITLE
Is the Windows version portable?

### DISCUSSION THREAD

**Discussion by 0x7FFFFFFFFFFFFFFF** [Eloston/ungoogled-chromium#1784] [Q&A] [answered]
I'm a new user and I downloaded the Windows version from https://github.com/Nifury/ungoogled-chromium-binaries/releases/download/96.0.4664.45-1/ungoogled-chromium_96.0.4664.45-1.1_windows.zip.
Is this version portable? I extracted the zip file and started using it without any issue. Thanks.

**Comment by networkException** [comment, marked as answer]
Please ask questions about binaries in the respective repositories, the ungoogled-chromium patchset itself does not define how chromium will get packaged.

**Reply by 0x7FFFFFFFFFFFFFFF** [reply]
Thanks.
